libraries/AP_OSD/AP_OSD.h

```diff
@@ -498,7 +500,19 @@ class AP_OSD
         uint16_t wp_number;
     };
 
+    struct StatsInfo {
```

Comment:
Would be nice to be able to send this stuff over mavlink at some stage, too!

yaapu GPS could potentially grab the same data in a post-flight screen.

So sometime in the future we might move this out into some other object... `AP_Stats` *maybe*....

Reply:
I wondered too why AP_Stats did not already have all this, anyway this is another PR's job